src/test/fuzz/coins_view.cpp

```diff
@@ -36,9 +36,7 @@ bool operator==(const Coin& a, const Coin& b)
 
 void initialize_coins_view()
 {
-    static const ECCVerifyHandle ecc_verify_handle;
-    ECC_Start();
-    SelectParams(CBaseChainParams::REGTEST);
+    static const auto testing_setup = MakeFuzzingContext<const TestingSetup>();
```

Comment:
why is it needed to spin up more than just ecc and params?

Reply:
Happy to delay this change, but `GetUTXOStats` calls `LookupBlockIndex` which asks `ChainstateManager` for its `BlockManager`. After `g_chainman` is de-globalized, we will no longer have an "implicitly-constructed-by-virtue-of-being-member-of-a-global" m_blockman, so we will need to run our initialization codepath at least until we have populated `NodeContext::chainman`.

However, it is not absolutely necessary right now, so I'd be happy to delay this change if you want to!

Reply:
Thanks, makes sense.